src/DataGridSample/MainWindow.axaml.cs

```diff
@@ -7,5 +8,8 @@ public partial class MainWindow : Window
     public MainWindow()
     {
         InitializeComponent();
```

Comment:
`AttachDevTools` should be automatically included with source generators enabled (which are by default).